airmozilla/main/static/main/js/calendar.js

```diff
@@ -1,31 +1,34 @@
 $(function() {
-    var weekends, firstDay = 0;
-    var x = new Date();
-    var currentTimeZoneOffsetInHours = x.getTimezoneOffset() / 60;
-    if (currentTimeZoneOffsetInHours <= -7) {
-        // sufficiently far east that some Friday events might actually
-        // start on a Saturday
-        weekends = true;
-        firstDay = 1;  // far-east people prefer Monday
-    } else {
-        weekends = false;
-    }
+    var weekends = true;
     var container = $('#calendar');
-    container.fullCalendar({
-        events: {
-            url: container.data('events-url'),
-            cache: true
-        },
-        timezone: 'local',
-        weekends: weekends,
-        firstDay: firstDay,
-        allDaySlot: false,
-        defaultTimedEventDuration: '01:00:00',
-        timeFormat: 'h:mm a',
-        header: {
-            left: 'prev,next today',
-            center: 'title',
-            right: 'month,agendaWeek,agendaDay'
-        }
+    var startsOnMonday = $('input[type="checkbox"]');
+    // if user isn't in American TZ then Monday is first day of week
```

Comment:
So this is a hack and we should explain that. We had a lengthy discussion about this on IRC. 
The conclusion was basically that we're "biased" against United States visitors who are in a "America/Something" time zone. That's because such a large majority of our visitors are from the US so we're given them the pleasure of the default matching their preference.